src/shogun2-web/src/main/java/de/terrestris/shogun2/web/AbstractLayerController.java

```diff
@@ -83,7 +82,7 @@ protected AbstractLayerController(Class<E> type) {
 	 */
 	@RequestMapping(value = "/setLayersForLayerGroup.action", method = RequestMethod.POST)
 	public @ResponseBody Map<String, Object> setLayersForLayerGroup(
-			Integer layerGroupId, @RequestParam("abstractLayerIds") List<String> abstractLayerIds) {
+			Integer layerGroupId, List<Integer> abstractLayerIds) {
```

Comment:
By inconsistent use I meant adding `@RequestParam` to all parameters :stuck_out_tongue_closed_eyes: